src/platform/Darwin/BleConnectionDelegateImpl.mm

```diff
@@ -103,7 +103,7 @@ - (void)removePeripheralsFromCache;
             // Make a copy of the device discriminator for the block to capture.
             SetupDiscriminator deviceDiscriminator = inDeviceDiscriminator;
 
-            ChipLogProgress(Ble, "%s", __FUNCTION__);
+            ChipLogProgress(Ble, "ConnectionDelegate NewConnection with discriminator");
```

Comment:
I wonder whether it's worth logging the discriminator.  Followup for that, if so.